scripts/install_build_tools.sh

```diff
@@ -24,11 +25,14 @@ echo "export PATH=/opt/cmake-${CMAKE_VERSION}-linux-x86_64/bin:$PATH" >> /root/.
 # shellcheck source=/dev/null
 export PATH=/opt/cmake-${CMAKE_VERSION}-linux-x86_64/bin:$PATH
 echo "CMake installation complete"
+# Installing Google Test
 GTEST_VERSION="1.14.0"
 echo "Installing Google Test ${GTEST_VERSION}" 
-git clone https://github.com/google/googletest.git -b "v{GTEST_VERSION}"
+git clone https://github.com/google/googletest.git -b "v${GTEST_VERSION}"
 cd googletest/
 cmake -Bbuild 
 cd build/
 make
 make install
+echo "Google Test installation complete"
```

Comment:
The script should remove the `googletest` repo after the build artifacts have been installed.

Reply:
good catch